---
permalink: /
title: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Throughout my career, I have extensively explored the fields of Bioinformatics, Cancer Genomics, Cancer Biology, multi-omics data analyses, and prediction modeling. My current research focuses on analyzing vast genomic datasets, including single-cell RNA-seq, bulk RNA-seq, ATAC-seq, Cut&Run-seq, Whole Exome Sequencing (WXS), and Whole Genome Sequencing (WGS), to uncover associations with clinical and phenotypic data of cancer patients.

I leverage a wide array of computational analysis tools and statistical programming languages, such as R, Perl, and Shell scripting within Unix/Linux server environments and High-Performance Computing (HPC) platforms. My expertise extends to accessing and utilizing various genomic databases, including ICGC, GDC, EGA, GDSC, CCLE, dbGaP, GEO, cBioPortal, and the Single Cell Atlas, to acquire and analyze data in formats like CEL, fastq, BAM, SAM, VCF, and more.
While I have not developed NGS-based software directly, I have significant experience in designing and optimizing such software for variant calling, detecting somatic mutations, gene fusions, gene rearrangements (structural variants), copy number variations (CNVs), and extracting gene expression data. I have integrated multi-omics data and applied them to therapeutic response prediction algorithms, machine learning models, and AI-based models (such as Autoencoders). 

My extensive knowledge and experience across various scientific disciplines enable me to collaborate effectively with experts in Bioinformatics, Cancer Genomics and Biology. This interdisciplinary approach allows me to successfully contribute to and fulfill research projects, driving forward our understanding and treatment of cancer.


Last update: June 2024
